arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/defer/DeferRegistry.java

```diff
@@ -224,4 +224,8 @@ public void resolveUponContextReEntry(Hub hub, CallFrame callFrame) {
   public void unscheduleForContextReEntry(ContextReEntryDefer defer, CallFrame callFrame) {
     contextReEntryDefers.get(callFrame).remove(defer);
   }
+
+  public void unscheduleForPostTransaction(PostTransactionDefer defer) {
+    postTransactionDefers.remove(defer);
+  }
```

Comment:
As explained below I'm not convinced we need this.

Reply:
see other comment. We schedule for postTx at the creation of the callSection, when the (modexp) subsection is still not created, so we don't have the info at the time of the schedulling.

Reply:
So wouldn't we have to unscheduled the postTransaction stuff, too ?

Reply:
The one we don't unSchedule (because it doesn't create NPE) is contextEntry